x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.test.tsx

```diff
@@ -113,12 +133,47 @@ describe('AlertSummaryView', () => {
       data: mockAlertDetailsData.map((item) => {
         if (item.category === 'event' && item.field === 'event.code') {
           return {
-            category: 'event',
-            field: 'event.code',
+            ...item,
             values: ['behavior'],
             originalValue: ['behavior'],
           };
         }
+        if (item.category === 'event' && item.field === 'event.category') {
+          return {
+            ...item,
+            values: ['malware', 'process', 'file'],
+            originalValue: ['malware', 'process', 'file'],
+          };
+        }
+        return item;
+      }) as TimelineEventsDetailsItem[],
+    };
+    const { container } = render(
+      <TestProvidersComponent>
+        <AlertSummaryView {...renderProps} />
+      </TestProvidersComponent>
+    );
+    expect(container.querySelector('div[data-test-subj="summary-view"]')).toMatchSnapshot();
```

Comment:
Same thing here. It might be a bit much to change all the ones in this file to inline-snapshots, but I think it's worth doing in the future